client/example_test.go

```diff
@@ -137,7 +137,7 @@ func ExampleKey_Attest() {
 	// On verifier, replay event log.
 	// TODO: decide which hash algorithm to use in the quotes. SHA1 is
 	// typically undesirable but is the only event log option on some distros.
-	_, err = server.ParseMachineState(attestation.EventLog, attestation.Quotes[0].Pcrs)
+	_, err = server.VerifyAttestation(attestation, server.VerifyOpts{Nonce: nonce})
```

Comment:
Since `VerifyAttestation` calls `VerifyQuote`, the code should probably be rewritten as follows:

```
	// TODO: establish trust in the AK (typically via an AK certificate signed
	// by the manufacturer).
	// On verifier, verify the Attestation message. This:
        // - checks the quote(s) against a stored public key/AK
	// certificate's public part and the expected nonce.
        // - replays the event log against the quoted PCRs
        // - extracts events into a MachineState message.
	// TODO: decide which hash algorithm to use in the quotes. SHA1 is
	// typically undesirable but is the only event log option on some distros.
	_, err = server.VerifyAttestation(attestation, server.VerifyOpts{Nonce: nonce, TrustedAKs: []crypto.PublicKey{ak.GetPublicKey()}})
```

and then remove lines 126 to 135 (calling internal.VerifyQuote and its comments).

Reply:
Done